<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTAndDRViewWidget</class>
 <widget class="QWidget" name="CTAndDRViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CTAndDRViewWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout_oriView">
         <property name="spacing">
          <number>1</number>
         </property>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_sliceView">
         <property name="spacing">
          <number>1</number>
         </property>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QScrollBar" name="verticalScrollBar_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout_3dView">
         <property name="spacing">
          <number>1</number>
         </property>
        </layout>
       </item>
       <item>
        <widget class="QScrollBar" name="horizontalScrollBar_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_result">
         <property name="spacing">
          <number>2</number>
         </property>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>170</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_topStatus_2">
    <property name="spacing">
     <number>2</number>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>170</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="horizontalLayout_BottomStatusBar_2">
    <property name="spacing">
     <number>2</number>
    </property>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
